ht-9/index.js

```diff
@@ -0,0 +1,25 @@
+function scrollspy () {
+  const scrollspyContent = document.querySelector(`[data-spy="scroll"]`);
+  const scrollspyLincksWraper = document.querySelector(scrollspyContent.dataset.target);
+  const scrollspyLincks = Array.from(scrollspyLincksWraper.querySelectorAll('.list-group-item'));
+  window.addEventListener('scroll', () => {
+    const scrollPosition = window.scrollY;
+    scrollspyLincks.forEach((linck, i, arr) => {
+      const listItem = document.querySelector(linck.getAttribute('href'));
+      const rect = listItem.getBoundingClientRect();
+      let remove = () => {
+        scrollspyLincks.forEach(linck => {
```

Comment:
instead of using forEach try to find only the active element and remove the 'active' class